src/cluster_migrateslots.c

```diff
@@ -353,6 +358,138 @@ void fireModuleSlotMigrationEvent(slotMigrationJob *job, int subevent) {
     zfree(info.slot_ranges);
 }
 
+sds clusterEncodeSlotImportsAuxField(int rdbflags) {
+    if (!server.cluster_enabled) return NULL;
+
+    /* Slot imports should be persisted on RDB files to allow replicas to PSYNC
+     * with their primary during an import and not expose in progress imports to
+     * users. AOF preambles cannot be used for resynchronization - so we skip
+     * those. */
+    if ((rdbflags & RDBFLAGS_AOF_PREAMBLE) != 0) return NULL;
+    sds s = NULL;
+
+    listNode *ln;
+    listIter li;
+    listRewind(server.cluster->slot_migration_jobs, &li);
+    while ((ln = listNext(&li)) != NULL) {
+        slotMigrationJob *job = ln->value;
+        if (isSlotMigrationJobFinished(job)) continue;
+        if (job->type != SLOT_MIGRATION_IMPORT) continue;
+        if (s == NULL) {
+            s = sdsempty();
+        } else {
+            sdscat(s, "/");
+        }
+        s = sdscatprintf(s, "%.40s:%.40s:", job->name, job->source_node_name);
+        sds slot_ranges = representSlotRangeList(job->slot_ranges);
+        s = sdscatsds(s, slot_ranges);
+        sdsfree(slot_ranges);
+    }
+    return s;
+}
+
+slotRange *decodeSlotRange(sds slot_range_str) {
```

Comment:
A small comment and `static` help understand the context and purpose of a function. (In general.)

```suggestion
/* Decode a part of an slot import RDB aux field.
 * Returns an allocated slotRange, or NULL on error. */
static slotRange *decodeSlotRange(sds slot_range_str) {
```

Reply:
Done